aws-auth-cognito/src/main/java/com/amplifyframework/statemachine/StateMachine.kt

```diff
@@ -24,9 +25,15 @@ import kotlinx.coroutines.Job
 import kotlinx.coroutines.launch
 import kotlinx.coroutines.newFixedThreadPoolContext
 
-internal typealias StateChangeListenerToken = UUID
 internal typealias OnSubscribedCallback = () -> Unit
 
+@JvmInline
```

Comment:
I don't think we need JVM annotations as all call sites are likely Kotlin.

Reply:
You need this since it's a `value` class

Reply:
Ah thanks, I forgot StateChangeListenerToken is still just a uuid